phpBB/styles/prosilver/theme/common.css

```diff
@@ -1269,6 +1269,10 @@ ul.linklist:after,
 	padding: 4px 6px;
 }
 
+.badge.hidden {
+	display: none;
+}
+
```

Comment:
If you prefer, it is possible to move the badge rule before the hidden rule, and that way, we would not need the extra rule. But I figured this would provide a hint to developers not to override the generic hidden rule (used elsewhere) but to modify this one, much more specific.